workers/unity/Packages/io.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Partials/Improbable.Gdk.TransformSynchronization.CompressedQuaternion

```diff
@@ -0,0 +1,215 @@
+private const float SqrtHalf = 0.70710678118f;
+
+public static readonly CompressedQuaternion Identity = FromUnityQuaternion(Quaternion.identity);
+
+public static bool operator ==(CompressedQuaternion a, CompressedQuaternion b) => a.Equals(b);
+public static bool operator !=(CompressedQuaternion a, CompressedQuaternion b) => !a.Equals(b);
+
+public override bool Equals(object obj)
+{
+    if (ReferenceEquals(null, obj))
+    {
+        return false;
+    }
+
+    return obj is CompressedQuaternion other && Equals(other);
+}
+
+public bool Equals(CompressedQuaternion other)
+{
+    return Data == other.Data;
+}
+
+public override int GetHashCode()
+{
+    return Data.GetHashCode();
+}
+
+/// <summary>
+///     Returns the string representation of the CompressedQuaternion.
+/// </summary>
+public override string ToString()
+{
+    return $"CompressedQuaternion{ToUnityQuaternion().ToString()}";
```

Comment:
This is missing the `( )` after `CompressedQuaternion`

Reply:
it's not, because the `Quaternion` `ToString` adds the `( )`:
```
({0:F1}, {1:F1}, {2:F1}, {3:F1})
```